service/src/main/java/uk/gov/hmcts/reform/fpl/service/HmctsCourtLookUpService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.base.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration.Court;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+@Service
+public class HmctsCourtLookUpService {
+
+    private final HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration;
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Autowired
+    public HmctsCourtLookUpService(HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration) {
+        this.hmctsCourtLookupConfiguration = hmctsCourtLookupConfiguration;
+    }
+
+    public Court getCourt(String localAuthorityCode) {
+        checkNotNull(localAuthorityCode, "Case does not have local authority assigned");
+
+        Court court = hmctsCourtLookupConfiguration.getLookupTable().get(localAuthorityCode);
+
+        if (Strings.isNullOrEmpty(court.getEmail())) {
```

Comment:
I feel that long term we wanna remove that validation in favour of failing fast at application lunch time when configuration is parsed.

Same way we might consider removing `HmctsCourtLookUpService` in favour of using `HmctsCourtLookupConfiguration` directly. We might hide `getLookupTable` method there in favour of having `getCourt` method.